### PR TITLE
build(aio): major cleanup of guides and dependencies

### PR DESCRIPTION
**Pull request by Foxandxss**
This cleanup old guides that are not shown, guides that are shown but are systemjs and removes a lot of shared dependencies that are not in use anymore.

I am sure that more can be deleted, but this is all I remember right now.

### CODE REVIEW

File: aio/content/navigation.json [hunk 1]
```diff
@@ -344,16 +344,6 @@
           "title": "Setup & Deployment",
```
Comment (gkalpak) [inline aio/content/navigation.json]: Maybe this title needs to change now. The guides below are not about setup any more (afaict).

File: aio/tools/examples/run-example-e2e.js [hunk 2]
```diff
@@ -127,12 +127,7 @@ function runE2eTestsSystemJS(appDir, outputFile) {
   const appBuildSpawnInfo = spawnExt('yarn', [config.build], { cwd: appDir });
   const appRunSpawnInfo = spawnExt('yarn', [config.run, '-s'], { cwd: appDir }, true);
 
-  let run = runProtractorSystemJS(appBuildSpawnInfo.promise, appDir, appRunSpawnInfo, outputFile);
-
-  if (fs.existsSync(appDir + '/aot/index.html')) {
-    run = run.then((ok) => ok && runProtractorAoT(appDir, outputFile));
-  }
-  return run;
+  return runProtractorSystemJS(appBuildSpawnInfo.promise, appDir, appRunSpawnInfo, outputFile);
```
Comment (gkalpak) [inline aio/tools/examples/run-example-e2e.js]: So, what does this change mean? Is aot not run any more?

File: aio/content/navigation.json (hunk 1 above)
Comment (Foxandxss) [reply aio/content/navigation.json]: I thought the same.

File: aio/tools/examples/run-example-e2e.js (hunk 2 above)
Comment (Foxandxss) [reply aio/tools/examples/run-example-e2e.js]: We don't have any systemjs + aot example to run anymore.

Comment (gkalpak) [reply aio/tools/examples/run-example-e2e.js]: I see. Do we have any cli + aot examples to run?

Comment (Foxandxss) [reply aio/tools/examples/run-example-e2e.js]: I think so, yes, but they run with `ng test` and not systemJS

**Comment by Foxandxss** [comment]
I realized that the ngUpgrade guide still uses the "setup" guide and the quickstart repo

What should we do?

**Comment by gkalpak** [comment]
Have someone rewrite the guides using the cli :grin:
<sub>(And making sure that "someone" is not me 😰)</sub>

**Comment by juleskremer** [comment]
@bradlygreen

File: aio/content/navigation.json (hunk 1 above)
Comment (gkalpak) [inline aio/content/navigation.json]: BTW, the [webpack entry][1] also needs to be removed.

[1]: https://github.com/IdeaBlade/angular/blob/011c989758805083afb3f5068c1bf55574adb9bc/aio/content/navigation.json#L64-L68

**Review by gkalpak — COMMENTED** [review]
WRT removing the webpack guide:
It would be nice to also incorporate https://github.com/angular/angular/pull/20213#issuecomment-344674300.

**Comment by IgorMinar** [comment]
I think that most of the PR is still relevant. Can someone confirm? @Foxandxss?

**Comment by Foxandxss** [comment]
@IgorMinar Yes, this is completely relevant. I hit a few decision issues but they were never resolved.

Some of the guides really need to go away but others depend on them.

I am open to do some hangout so I could explain what is going on, take a decision and then I can rebase this.

**Comment by jenniferfell** [comment]
@jbogarthyde Some of this might be useful in your work on setup and deployment docs.

**Comment by petebacondarwin** [comment]
@jenniferfell should we just close this? the content will still be in GitHub if you want to come back and extract it.

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
This should be closed. It is out of date and very conflicted. I don't believe that keeping it around is encouraging any work on it to be prioritized. Such work should be captured in a Jira ticket and allocated appropriately.

**Pull request closed without merging**